app/javascript/styles/mastodon/variables.scss

```diff
@@ -111,6 +111,8 @@ $font-monospace: 'mastodon-font-monospace' !default;
   --surface-variant-active-background-color: #{lighten($ui-base-color, 4%)};
   --on-surface-color: #{transparentize($ui-base-color, 0.5)};
   --avatar-border-radius: 8px;
+  --media-outline-color: #{rgba(#fcf8ff, 0.15)};
```

Comment:
Not a blocker, but the way this color is defined is a bit inconsistent with how every other one is defined.

Reply:
It's a new color from Figma and a bit of improvisation from me.